test/document.spec.js

```diff
@@ -546,21 +546,31 @@ describe('Document', () => {
 
   describe('Lock Status', () => {
     describe('#lock', () => {
-      it('should lock the document', () => (
+      const lockDocument = () => {
         repository.fetch(FILE_TEST_PATH)
           .then((doc) => doc.lock())
           .then((doc) => doc.fetchLockStatus())
           .then((status) => {
             expect(status.lockOwner).to.exist();
             expect(status.lockCreated).to.exist();
-          })
-      ));
+          });
+      };
+
+      it('should lock the document', lockDocument);
+
+      it('should lock the locked document if the user is the lock owner', lockDocument);
```

Comment:
Technically the document is no re-locked. Maybe "should not fail if the user is the lock owner"?

Reply:
yes i agree, for be more clear wdyt  about a message like  "should call the lock operation on a locked document if the user is the lock owner" the idea is to understand more easily the purpose of the test :)

Reply:
I agree with @ataillefer for "should not fail if the user is the lock owner". You will end up in the test logs with:
```
Lock Status > #lock > should not fail if the user is the lock owner
```
Which is quite clear.

"the lock operation" is an implementation detail of the `#lock` method.

Reply:
Well, what you suggest is not clear to me. You are calling the lock operation anyway. I'm more in favor of a user-friendly description than a technical one. But maybe you just want to add the fact that the *document is locked*?  So "should not fail on a locked document if the user is the lock owner"?
@troger WDYT?